stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1225,6 +1234,12 @@ public final class TestSuite {
       f()
     }
     let test = _testByName(testName)
+
+#if SWIFT_RUNTIME_ENABLE_LEAK_CHECKER
+    let nametag = name
+    startTrackingObjects(UnsafeMutablePointer(nametag._core.startASCII))
```

Comment:
Or rather, I am worried that if we use the closure, allocations may occur that the leak detector may catch. On the other hand, allocating statically before/after (perhaps using UnsafeMutablePointer, will ensure that we do not need to worry about unforeseen allocations coming up.